frontend/src/forms/AdminForm.jsx

```diff
@@ -36,6 +36,10 @@ export default function AdminForm({ isUpdateForm = false }) {
             required: true,
             message: 'Please input your Email!',
```

Comment:
I think the name we use for `E-mail` should be consistent. I see the label is called `E-mail` so I think this should change to `Please input your E-mail!`